.benchmarks/gobenchdata-checks.yml

```diff
@@ -0,0 +1,25 @@
+checks:
+    - name: VM checks
+      description: |-
+        It checks performance regressions on VM package.
+      package: "gnovm"
+      benchmarks: []
+      diff: (current.NsPerOp - base.NsPerOp) / base.NsPerOp * 100
+      thresholds:
+        max: 10
```

Comment:
what it means, and what happens if we hit this?

Reply:
example check output report: https://github.com/ajnavarro/gno/actions/runs/5280312972/jobs/9553479785#step:4:63